WCFDataService/Client/System/Data/Services/Client/ClientEdmModel.cs

```diff
@@ -443,7 +447,8 @@ private EdmTypeCacheValue GetOrCreateEdmTypeInternal(IEdmStructuredType edmBaseT
                             List<IEdmProperty> loadedProperties = new List<IEdmProperty>();
                             foreach (PropertyInfo property in ClientTypeUtil.GetPropertiesOnType(type, /*declaredOnly*/edmBaseType != null).OrderBy(p => p.Name))
                             {
-                                IEdmProperty edmProperty = this.CreateEdmProperty(complexType, property);
+                                FieldInfo backingField = FindBackingField(property, this.ProxyBackingFieldNamingConvention);
```

Comment:
In V3, we do not support navigation on complex. And for complex itself, lazy initialization seems not quite necessary. What do you think?